typescript/db/src/repository/image-processing.ts

```diff
@@ -81,12 +83,34 @@ const generateThumbnail = async ({
   }
 };
 
+const shouldNotLoadImage = ({
+  width,
+  height,
+  mimetype,
+  url,
+  thumbnail20Url,
+  thumbnail50Url,
+  thumbnail100Url,
+  thumbnail200Url,
+  thumbnail500Url,
+}: Parameters<Repository["imageProcessing"]["processImage"]>[0]): boolean => {
+  return (
+    [width, height, mimetype, url].every((value) => !isEmpty(value)) &&
+    [
+      thumbnail20Url,
+      thumbnail50Url,
+      thumbnail100Url,
+      thumbnail200Url,
+      thumbnail500Url,
+    ].some((value) => !isEmpty(value))
```

Comment:
why "some"?

Reply:
Why not ?

Reply:
It's a way to say "if there is a thumbnail, don't load the image"
If you want more info, ask @bbenoist  as he introduce this part of the code